nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -35,8 +35,15 @@ with lib;
   # Enable sound in graphical iso's.
   hardware.pulseaudio.enable = true;
 
-  # Spice guest additions
-  services.spice-vdagentd.enable = true;
+  # VM guest additions to improve host-guest interaction
+  services.spice-vdagentd.enable = mkDefault true;
```

Comment:
the other options don't use `mkDefault`, probably because this config is not intended to be used outside of the installer image. it should be consistent. i suggest to remove it